tests/callbacks/test_data_monitor.py

```diff
@@ -38,45 +55,71 @@ def test_base_log_interval_override(log_histogram, tmpdir, log_every_n_steps, ma
     ],
 )
 @mock.patch("pl_bolts.callbacks.data_monitor.DataMonitorBase.log_histogram")
-def test_base_log_interval_fallback(log_histogram, tmpdir, log_every_n_steps, max_steps, expected_calls, datadir):
+def test_base_log_interval_fallback(
+    log_histogram,
+    tmpdir,
+    log_every_n_steps,
+    max_steps,
+    expected_calls,
+    datadir,
+    catch_warnings,
+):
     """Test that if log_every_n_steps not set in the callback, fallback to what is defined in the Trainer."""
+    warnings.filterwarnings(
```

Comment:
why is this filtering necessary?

Reply:
So, the idea is that all reviewed tests are having this fixture called `catch_warnings`, which essentially turns warnings into errors. Ideally, we don't want any warnings as that may indicate a compatibility or deprecation problem, however there are cases in which the warning is harmless - for example here. We don't need to pass multiple workers in a test in order to see it works, because that's not what we are testing, but PL will still raise a warning. And therefore, we're filtering it out here